api/command_test.go

```diff
@@ -256,9 +257,22 @@ func TestTestCommand(t *testing.T) {
 
 	time.Sleep(100 * time.Millisecond)
 
-	p1 := Client.Must(Client.GetPosts(channel1.Id, 0, 2, "")).Data.(*model.PostList)
-	if len(p1.Order) != 1 {
-		t.Fatal("Test command failed to send")
+	p1 := Client.Must(Client.GetPosts(channel1.Id, 0, 10, "")).Data.(*model.PostList)
+	// including system 'joined' message
+	if len(p1.Order) != 2 {
+		t.Fatal("Test command response failed to send")
+	}
+
+	cmdPosted := false
+	for _, post := range p1.Posts {
+		if strings.Contains(post.Message, "test command response") {
```

Comment:
Not sure what the standard is, but should "test command response" be extracted to a constant since it's used in two places?

Reply:
These are hardcoded as well from the api itself (v3/v4); here's from v4 - https://github.com/mattermost/platform/blob/213a072b38d29d3c3ec8e150584685b1144a7d6a/api4/command.go#L306
Also not sure, but since it's used for testing only, I'm leaning towards not adding constant for that purpose.